palantiri/core/datahandler.py

```diff
@@ -88,21 +89,22 @@ def __repr__(self):
 
     def dump(self, to_insert):
         cur = self.conn.cursor()
-        cur.executemany(("INSERT INTO page(url, content, datescraped, status_code, headers) "
-                         "VALUES (%s, %s, %s, %s, %s) "
-                         "ON CONFLICT (url) DO NOTHING;"),
+        logging.info("************* Saving %d pages to DB" % len(to_insert))
+        cur.executemany(("INSERT INTO page(url, content, datescraped, CrawlerId) "
+                         "VALUES (%s, %s, %s, %s) ;"),
```

Comment:
I think we should have an `ON CONFLICT`. Is there a reason that was removed?

Reply:
Similar comment of just making code work with the existing database structure.